torch/_subclasses/fake_tensor.py

```diff
@@ -1031,8 +1033,22 @@ def should_allow_numbers_as_tensors(func: OpOverload):
     )
 
 
+def is_fbcode():
+    return not hasattr(torch.version, "git_version")
+
+
 class FakeTensorConfig:
-    debug = os.environ.get("TORCH_FAKE_TENSOR_DEBUG", False)
+    debug = os.environ.get("TORCH_FAKE_TENSOR_DEBUG", "0") == "1"
+    cache_enabled = (
+        os.environ.get(
+            "TORCH_FAKE_TENSOR_DISPATCH_CACHE",
+            "0" if is_fbcode() else "1",
+        )
+        == "1"
+    )
+    cache_crosscheck_enabled = (
+        os.environ.get("TORCH_FAKE_TENSOR_DISPATCH_CACHE_CROSSCHECK", "0") == "1"
+    )
```

Comment:
`debug` and `cache_crosscheck_enabled` make sense to put here but I think `cache_enabled` we should move to a more standardized place. `torch/_functorch/config.py` or `torch/_dynamo/config.py`. Just to standardize config options which affect runtime behavior. Debug options I think is okay to be a little bit more scattered.